x-pack/platform/plugins/private/data_visualizer/public/lite/file_status.tsx

```diff
@@ -59,29 +81,42 @@ export const FileStatus: FC<Props> = ({ fileStatus, uploadStatus, deleteFile, in
         ) : (
           <>
             <EuiFlexGroup gutterSize="s" alignItems="center">
-              <EuiFlexItem grow={8}>
-                <EuiText size="xs">
-                  <span css={{ fontWeight: 'bold' }}>{fileStatus.fileName}</span>{' '}
-                  <span>{fileStatus.fileSize}</span>
-                </EuiText>
-              </EuiFlexItem>
+              {showFileContentPreview || showFileSummary ? (
+                <EuiFlexItem grow={false}>
+                  <EuiButtonIcon
+                    iconType={expanded ? 'arrowDown' : 'arrowRight'}
```

Comment:
Did you consider using `EuiAccordion` instead?

Reply:
It didn't fit with the style of the component and the other items which are displayed.